### DISCUSSION TITLE
Install applications somewhere other than /Applications/

### DISCUSSION THREAD

**Discussion by csterritt** [Homebrew/discussions#3285] [Everyday usage] [answered]
Hi - sorry if this has been asked before, it's kind of hard to search for.

I went to install an `meld`, and its last step is to install the application in /Applications/, which of course requires an admin. So, of course, it asked for my password (via `sudo`, I imagine).

(A) I run as a least-privileged user, and would like to continue to do so.
(B) I install applications in /Users/chris/Applications/, which I do have write privileges to.

Is there a way to ask homebrew to use that directory instead?

Thanks!

**Comment by SMillerDev** [comment, marked as answer]
https://docs.brew.sh/Manpage#global-cask-options

**Reply by csterritt** [reply]
Thanks so much -- sorry I missed that!
